content/en/docs/languages/js/instrumentation.md

```diff
@@ -1213,9 +1223,14 @@ For more on synchronous and asynchronous instruments, and which kind is best
 suited for your use case, see
 [Supplementary Guidelines](/docs/specs/otel/metrics/supplementary-guidelines/).
```

Comment:
Could we consolidate this sentence with the alert you added just below. It doesn't make sense to have a normal paragraph refer the reader to the spec via: "For more on synchronous and asynchronous instruments, and which kind is best suited for your use case, see [Supplementary Guidelines](/docs/specs/otel/metrics/supplementary-guidelines/)" immediately followed by an alert that says ~ "for more info see the concept page".

Reply:
ah, oh, I didn't see that. This may also fix some of the things above.